sunpy/map/sources/hinode.py

```diff
@@ -98,6 +98,18 @@ def processing_level(self):
             return
         return int(lvl)
 
+    @property
+    def unit(self):
+        # XRT data values are in DN and are converted into DN/s if the data has been normalized.
+        # A tag starting with "XRT_RENORMALIZE" is added to the HISTORY tag in that case.
+        # See Table 1.1 and Section 2.11 of the XRT Analysis Guide.
+        unit = super().unit
+        if not unit:
+            unit = u.ct
+            if "XRT_RENORMALIZE" in self.meta.get('HISTORY'):
```

Comment:
```suggestion
            history = self.meta.get('HISTORY', '')
            if "xrt_renormalize" in history.lower():
```

Reply:
To protect against any capitalization issues. Sorry, should've caught that in my previous suggestion.